rosdep/base.yaml

```diff
@@ -823,6 +823,14 @@ geographiclib:
     vivid: [libgeographic-dev]
     wily: [libgeographic-dev]
     xenial: [libgeographic-dev]
+geographiclib-tools:
+  debian: [geographiclib-tools]
+  ubuntu:
+    precise: [geographiclib-tools]
```

Comment:
Not really. Just wanna to complete OS dependencies as much as possible. Should I take it then?